crowbar_framework/spec/lib/remote_node_spec.rb

```diff
@@ -20,37 +20,39 @@
   context "reboot_done?" do
     it "should return true because boottime is higher than reboot requesttime" do
       RemoteNode.stubs(:ssh_cmd_get_boottime).returns(100).once
-      RemoteNode.reboot_done?("localhost", 1).should eql true
+      expect(RemoteNode.reboot_done?("localhost", 1)).to eql true
     end
 
     it "should return false because boottime is lower than reboot requesttime" do
       RemoteNode.stubs(:ssh_cmd_get_boottime).returns(100).once
-      RemoteNode.reboot_done?("localhost", 999).should eql false
+      expect(RemoteNode.reboot_done?("localhost", 999)).to eql false
     end
 
     it "should return false because boottime equal to reboot requesttime" do
       RemoteNode.stubs(:ssh_cmd_get_boottime).returns(10).once
-      RemoteNode.reboot_done?("localhost", 10).should eql false
+      expect(RemoteNode.reboot_done?("localhost", 10)).to eql false
     end
 
     it "should return true because reboot requesttime is unknown" do
-      RemoteNode.reboot_done?("localhost", 0).should eql true
+      expect(RemoteNode.reboot_done?("localhost", 0)).to eql true
     end
   end
   context "ssh get boottime" do
     it "should handle unsucessful ssh command" do
+      logger = mock('Logger')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.